src/parser/jobs/blm/modules/Triplecast.tsx

```diff
@@ -0,0 +1,50 @@
+import {Plural, Trans} from '@lingui/macro'
+import {DataLink} from 'components/ui/DbLink'
+import {Event} from 'event'
+import {Analyser} from 'parser/core/Analyser'
+import {filter} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import {Actors} from 'parser/core/modules/Actors'
+import {Data} from 'parser/core/modules/Data'
+import Suggestions, {SEVERITY, TieredSuggestion} from 'parser/core/modules/Suggestions'
+import React from 'react'
+
+export class Triplecast extends Analyser {
+	static override handle = 'triplecast'
+
+	@dependency private actors!: Actors
+	@dependency private data!: Data
+	@dependency private suggestions!: Suggestions
+
+	private overwrittenTriples = 0
+
+	override initialise() {
+		this.addEventHook(filter<Event>().source(this.parser.actor.id).action(this.data.actions.TRIPLECAST.id), this.onTriplecast)
+		this.addEventHook('complete', this.onComplete)
+	}
+
+	private onTriplecast() {
+		if (this.actors.current.hasStatus(this.data.statuses.TRIPLECAST.id)) {
+			this.overwrittenTriples++
+		}
+	}
+
+	private onComplete() {
+		// Suggestion not to overwrite Sharpcast
+		this.suggestions.add(new TieredSuggestion({
+			icon: this.data.actions.SHARPCAST.icon,
+			content: <Trans id="blm.triplecast.suggestions.overwrote-triplecasts.content">
+				You lost at least one instant cast spell by using <DataLink action="TRIPLECAST" /> while the status was already active.
+			</Trans>,
+			tiers: {
+				1: SEVERITY.MINOR,
+				3: SEVERITY.MEDIUM,
+				5: SEVERITY.MAJOR,
+			},
```

Comment:
Same as comment on Sharpcast.

Reply:
This one's probably worth making tighter, since Triplecast is more of a DPS increase due to Fire 4's longer-than-the-GCD cast time, and the loss of mobility. Dropped the Medium/Major tiers down by 1 so it's a 1/2/4 tier.